tools/net/simple_file_transfer.py

```diff
@@ -0,0 +1,96 @@
+#!/usr/bin/env python3
+############################################################################
+#
+# Copyright 2020 Samsung Electronics All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+# either express or implied. See the License for the specific
+# language governing permissions and limitations under the License.
+#
+############################################################################
```

Comment:
Could you leave a line's space after the license please?

Reply:
it's exactly same to hello_main.c. I don't understand.

Reply:
Between lines 18 and 19 please.